docs/docs/naming.md

```diff
@@ -4,7 +4,7 @@ title: Service Naming
 
 # Service Naming
 
-Mesos-DNS defines a DNS domain for Mesos tasks (default `.mesos`, see [instructions on configuration](configuration-parameters.html)). Running tasks can be discovered by looking up A and, optionally, SRV records within the Mesos domain. 
+Mesos-DNS defines a DNS domain for Mesos tasks (default `.mesos`, see [instructions on configuration](configuration-parameters.html)). Running tasks can be discovered by looking up A and, optionally, SRV records within the Mesos domain.
```

Comment:
There are lots of whitespace changes here which seem unrelated to the goal of "numbered ports". Docs improvements are welcome, but such changes here that are unrelated to the overall goal end up making this harder to review.